### DISCUSSION TITLE
Setup hack with multiple SSIDs

### DISCUSSION THREAD

**Discussion by caykroyd1** [roleoroleo/yi-hack-Allwinner-v2#898] [Ideas]
Is there a way already or a possible feature to add multiple SSIDs and passwords to the configure_wifi.cfg file?  I have a camera that I want to be mobile between two houses and don't want to change wife each time.

**Comment by Dotatay** [comment]
I think I read that only one SSID works.

However, I have recently noticed that wpa_cli is available. You may be able to try changing the WiFi network via SSH.
Another option would be to place a wpa_supplicant.conf file in the factory folder and possibly automate the process of changing credentials.
